### PR TITLE
System Configuration and Scripts

### PR DESCRIPTION
**Pull request by jpdagostino**
Make it easier for people to host mafia. Included install scripts, service files, nginx config files, and an update script

### CODE REVIEW

File: system/install.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
+cd /
+echo Make sure you run this as root!
```
Comment (Jack-Papel) [inline system/install.sh:2]: Consider: https://unix.stackexchange.com/questions/28791/prompt-for-sudo-password-and-programmatically-elevate-privilege-in-bash-script
Or aborting if not root

File: system/update-rootless.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
+cd /mafia
+
+echo Pulling Updates
+git stash
```
Comment (Jack-Papel) [inline system/update-rootless.sh:4]: git stash without git stash pop? Seems like a mistake. Disregard if not though

**Review by Jack-Papel — APPROVED** [review]
Address comments, or don't.

File: system/update-rootless.sh (hunk 2 above)
Comment (jpdagostino) [reply system/update-rootless.sh:4]: That makes more sense! Before I was copying to the system-config directory that was added to the gitignore.

**Pull request merged**